src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/Extensions.cs

```diff
@@ -60,15 +63,15 @@ public static TypeDefinition GetType (this LinkContext context, string assemblyN
 			return ad == null ? null : GetType (ad, typeName);
 		}
 
-		public static MethodDefinition GetMethod (this LinkContext context, string ns, string typeName, string name, string [] parameters)
+		public static MethodDefinition GetMethod (this LinkContext context, string assemblyName, string typeName, string name, string [] parameters)
 		{
-			var type = context.GetType (ns, typeName);
+			Console.WriteLine ($"assembly: {assemblyName}");
```

Comment:
Is this `Console.WriteLine()` leftover from debugging?

Reply:
Indeed, thanks!